pxtlib/tutorial.ts

```diff
@@ -282,6 +290,23 @@ ${code}
         return { header, hint };
     }
 
+    /* Parse Tutorial Validation Rules */
+    function parseTutorialValidationRules(body: string, tutorialCodeValidation?: boolean): pxt.Map<boolean> {
```

Comment:
(late comment/this can be in a separate PR) i think this should use JSON.parse instead of manually parsing (see https://github.com/microsoft/pxt/blob/master/pxtlib/util.ts#L276)